assets/html/error-pages/service-unavailable.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>Service unavailable - Office for National Statistics</title>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1"/>
+    <meta charset="utf-8"/>
+    <meta content="width=device-width,initial-scale=1.0,user-scalable=1" name="viewport">
+    <meta name="format-detection" content="telephone=no">
+    <meta name="theme-color" content="#58595B">
+    <meta name="apple-mobile-web-app-status-bar-style" content="#58595B">
+    <link rel="shortcut icon" href="https://cdn.ons.gov.uk/assets/images/favicon.ico" type="image/x-icon">
+
+    <!--[if IE]><![endif]-->
+    <!--[if lte IE 8]>
+    <link rel="stylesheet" href="https://cdn.ons.gov.uk/sixteens/b6c1f18/css/old-ie.css">
+    <![endif]-->
+    <!--[if IE 9]>
+    <link rel="stylesheet" href="https://cdn.ons.gov.uk/sixteens/b6c1f18/css/ie-9.css">
+    <![endif]-->
+    <!--[if gt IE 9]><!-->
+    <link rel="stylesheet" href="https://cdn.ons.gov.uk/sixteens/b6c1f18/css/main.css">
+    <!--<![endif]-->
+</head>
+<body>
+<header class="border-bottom--iron-sm border-bottom--iron-md">
+    <div class="wrapper">
+        <div class="header col-wrap">
+            <div class="col col--lg-one-third col--md-one-third">
+                <a id="logo-link" href="/">
+                    <!--[if lte IE 8]>
+                    <img class="logo" src="https://cdn.ons.gov.uk/assets/images/ons-logo/v2/ons-logo.png"
+                         alt="Office for National Statistics">
+                    <![endif]-->
+                    <!--[if gte IE 9]><!-->
+                    <img class="logo" src="https://cdn.ons.gov.uk/assets/images/ons-logo/v2/ons-logo.svg"
+                         alt="Office for National Statistics">
+                    <!--<![endif]-->
+                </a>
+            </div>
+        </div>
+    </div>
+</header>
+
+<main class="main main--sticky-footer-offset">
+    <div class="page-header slate">
+        <div class="wrapper">
+            <h1 class="break-word margin-top-sm--3 margin-bottom-sm--3"><strong>Sorry, this website is unavailable</strong></h1>
+        </div>
+    </div>
+    <div class="wrapper panel--mar">
+        <div class="grid-wrap">
+            <div class="grid-col desktop-grid-full-width">
+                <article>
+                    <div class="box box--padded box--red-dark box--red-dark--separated-left sectioned">
+                        <div class="grid-wrap">
+                            <div class="grid-col desktop-grid-two-thirds adjust-font-size--16">
+                                <p>The ONS Digital blog is unavailable while we make them accessible for our users.</p>
```

Comment:
Not an issue with anything you've done, but should this be "The ONS Digital blog is unavailable while we make **it** accessible for all our users"? Just checking as we've removed the reference to the developer site now